From d6ef0fd66c42ffbba13b5d7a006457ca7f27fb44 Mon Sep 17 00:00:00 2001
From: Gyeongjae Choi <def6488@gmail.com>
Date: Fri, 2 Sep 2022 04:25:31 +0000
Subject: [PATCH] Hardcode float precision

HDF5 tries to check float precision by compiling and running
a simple C program but it is not available with SIDE_MODULE setting.
So we hardcode the value.
---
 config/cmake/ConfigureChecks.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config/cmake/ConfigureChecks.cmake b/config/cmake/ConfigureChecks.cmake
index 902ddd3d3e..0a92cbb9ba 100644
--- a/config/cmake/ConfigureChecks.cmake
+++ b/config/cmake/ConfigureChecks.cmake
@@ -331,7 +331,8 @@ set (PROG_SRC
 )
 
 C_RUN ("maximum decimal precision for C" ${PROG_SRC} PROG_RES)
-file (READ "${RUN_OUTPUT_PATH_DEFAULT}/pac_Cconftest.out" PROG_OUTPUT4)
+list(APPEND PROG_OUTPUT4 36)
+list(APPEND PROG_OUTPUT4 0)
 message (STATUS "Testing maximum decimal precision for C - ${PROG_OUTPUT4}")
 
 # dnl The output from the above program will be:
-- 
2.37.0

